setup.py

```diff
@@ -1,54 +1,48 @@
 # -*- coding: utf-8 -*-
-import ast
 import os
+
 from setuptools import setup, find_packages
 
+from grader import (
```

Comment:
Are we sure that we'll always import the right `grader`? E.g. if the user already installed grader in some previous version, will the one from cwd "win"?

Reply:
Yes, cwd is the first entry in `sys.path` by default. Test case:

```console
 ~/projects/play/import-order
 $ mkdir numpy
(36)
jni@molino Mon Nov 13 11:41
 ~/projects/play/import-order
 $ echo import os > numpy/__init__.py
(36)
jni@molino Mon Nov 13 11:41
 ~/projects/play/import-order
 $ cat numpy/__init__.py
import os
(36)
jni@molino Mon Nov 13 11:41
 ~/projects/play/import-order
 $ python
Python 3.6.1 |Anaconda custom (x86_64)| (default, May 11 2017, 13:04:09)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> dir(np)
['__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', 'os']
```

Reply:
Indeed. And people almost always run setup.py in the same way: by `cd`-ing into the directory and then calling `python setup.py ...`. So OK.